docs/cugraph/source/conf.py

```diff
@@ -205,4 +206,10 @@ def setup(app):
 # The following is used by sphinx.ext.linkcode to provide links to github
 linkcode_resolve = make_linkcode_resolve(
     "https://github.com/rapidsai/cugraph/blob/{revision}/python/{path}#L{lineno}"
-)
\ No newline at end of file
+)
+
+breathe_projects = {
+    'libcugraphops': '/tmp/xml_tar/libcugraphops',
+    'libwholegraph': '/tmp/xml_tar/libwholegraph'
```

Comment:
```suggestion
    'libcugraphops': os.environ['XML_DIR_LIBCUGRAPHOPS'],
    'libwholegraph': os.environ['XML_DIR_LIBWHOLEGRAPH']
```

This change allows us to avoid hardcoding absolute paths.

The Python script will exit with an error code if either of these environment variables don't exist.